List/Iconic/index.jsx

```diff
@@ -25,17 +25,20 @@ Wrapper.displayName = 'List.Iconic.Wrapper'
 export function Item ({className, icon, children, styles, ...props}) {
   const classNames = classNamesBind.bind({...defaultStyles, ...styles})
 
-  return <li className={classNames(classes.item, className)} {...props}>
-    <div className={classNames(classes.itemIcon)}>
-      {icon}
-    </div>
-
-    <div className={classNames(classes.content)}>
-      <Paragraph.Secondary>
-        {children}
-      </Paragraph.Secondary>
-    </div>
-  </li>
+  return <div className={classNames(classes.item)} {...props}>
+    <tbody>
```

Comment:
Just to ease my conscience, I need to again mention that you can do this with just CSS, instead of putting tables in.

Conceptually:
```
<div className={classNames(classes.item)} {...props}>
  <div style="display: table-row;">
    <div className={classNames(classes.itemIcon)} style="display: table-cell;">
      {icon}
    </div>
    <div style="display: table-cell;">
      <Paragraph.Secondary>{children}</Paragraph.Secondary>
    </div>
  </div>
</div>
```